### PR TITLE
Zend Framework 3.0.0

### PR DESCRIPTION
**Pull request by FabianKoestring**
- Declaration of Listeners `attach` must be compatible to `ListenerAggregateInterface->attach(events: Zend\EventManager\EventManagerInterface, [priority: int=1)`.
- Fixed namespace `Zend\Mvc\Router\RouteMatch` to `Zend\Router\RouteMatch`.
- Removed assertion of attached or detached listeners. http://zend-eventmanager.readthedocs.io/en/latest/migration/removed/#eventmanagerinterfacegetevents-and-getlisteners

ping @neeckeloo

### CODE REVIEW

**Comment by neeckeloo** [comment]
You have to remove php 5.5 from travis file to comply with zf3 requirements.

**Comment by FabianKoestring** [comment]
> You have to remove php 5.5 from travis file to comply with zf3 requirements.

Done!

File: composer.json [hunk 1]
```diff
@@ -16,13 +16,13 @@
     "require": {
         "php": "^5.5 || ^7.0",
```
Comment (neeckeloo) [inline composer.json]: You can also update package requirements : ```^5.6 || ^7.0```

**Comment by FabianKoestring** [comment]
:+1:
@neeckeloo - Will you tag a new release?

**Comment by neeckeloo** [comment]
I tagged a new release 2.0 according to breaking changes.

**Pull request merged**